CMakeLists.txt

```diff
@@ -162,16 +185,28 @@ target_sources(unassemblize PRIVATE
     src/workqueue.h
     src/imguiclient/imguiapp.cpp
     src/imguiclient/imguiapp.h
-    src/imguiclient/imguiwin32.cpp
-    src/imguiclient/imguiwin32.h
     src/imguiclient/utility/imgui_misc.cpp
     src/imguiclient/utility/imgui_misc.h
     src/imguiclient/utility/imgui_scoped.h
     src/imguiclient/utility/imgui_text_filter.cpp
     src/imguiclient/utility/imgui_text_filter.h
 )
 
+if(WINDOWS)
+    target_sources(unassemblize PRIVATE
+        src/imguiclient/imguiwin32.cpp
+        src/imguiclient/imguiwin32.h
+    )
+else()
+    target_sources(unassemblize PRIVATE
+        src/imguiclient/imguiglfw.cpp
+        src/imguiclient/imguiglfw.h
+    )
+endif()
+
 target_link_libraries(unassemblize PRIVATE
+    imgui_unix
```

Comment:
Should this be excluded from Windows?

Reply:
Yes, I updated cmakelists to keep the non-windows items separate .